Formula/r/rtags.rb

```diff
@@ -58,7 +50,8 @@ class Rtags < Formula
   uses_from_macos "zlib"
 
   def install
-    system "cmake", "-S", ".", "-B", "build", "-DRTAGS_NO_BUILD_CLANG=ON", *std_cmake_args
+    # Fix to add backward compatibility for CMake 
+    system "cmake", "-S", ".", "-B", "build", "-DCMAKE_POLICY_VERSION_MINIMUM=3.5", *std_cmake_args
```

Comment:
better to upstream the change

Reply:
master and tag branch is differ too many and patch is not applied well.

Reply:
yeah, indeed, that is why I resolved the thread :)

maybe keep the code and just add a PR ref in case we can remove next time. what do you think?